src/angular/2-building-first-app/building-first-app.md

```diff
@@ -221,7 +182,7 @@ This file is the root module for our app. Every Angular app has at least one mod
 - **declarations** `[array]`: where we include components, directives, and pipes that will be used by this module.
 - **imports** `[array]`: where we include any other modules our app needs to use. This may include 3rd party modules like bootstrap datepickers, or modules we’ve created.
 - **providers** `[array]`: where we include services that we want used at the global app level
-- **bootstrap** `[array]`: where we include the root AppModule - this is the main Application view that hosts all of our other app views.
+- **bootstrap** `[array]`: where we include the root `AppComponent` - this is the main Application view that hosts all of our other app views.
```

Comment:
@fabioemoutinho I wrote this as something to fix, is this right?

Reply:
correct
![image](https://github.com/bitovi/academy/assets/26363255/424393c0-0b62-49c6-b05c-d20b8ee49837)